layouts/event/single.html

```diff
@@ -96,3 +101,4 @@
 }
 </script>
 {{ end }}
+{{ with .Params.notices }}<p>{{ . | markdownify }}</p>
```

Comment:
Please keep the (human-readable) content within the `<article>` tag and above the comments. I am sorry if I was unclear, I was talking about the bottom of the page, not the bottom of the file.

Also, you are missing an `{{ end }}` tag here.